apps/web/src/modules/dao/components/DaoFeed/DaoFeed.tsx

```diff
@@ -59,7 +63,7 @@ export const DaoFeed = () => {
   if (!isLoading && !hasError && !hasThreeDaos) return null
 
   return (
-    <RecentlyCreated>
+    <RecentlyCreated isDashboard={isDashboard}>
```

Comment:
I think introducing this boolean is creating some unnecessary complications -- Not sure if this component should be reusued in the way it is -- there is very little shared between the two uses DaoFeed and Dashboard, except the Explore All button, which I think can be refactored and reusued in two separate components

would prefer if we wrote a separate router for the dashboard case and avoid introducing this `isDashboard` prop

Reply:
Hmm... Not sure what you mean by this one. 

I get that it's definitely adding some complexity, but the DAO feed is doing a lot more than displaying the Explore All button. I definitely didn't like the pattern, but assumed it would be better than having two major components with very, very similar code. 

Also, I'm trying to figure out what you mean by router in this case.

Reply:
I'm still down to refactor this to its own component, btw. 

Just want to make sure I'm understanding you correctly.